pom.xml

```diff
@@ -883,7 +883,12 @@
       <dependency>
         <groupId>org.bouncycastle</groupId>
         <artifactId>bcprov-jdk15on</artifactId>
-        <version>${bcprov-jdk15on.version}</version>
+        <version>${bouncycastle.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.bouncycastle</groupId>
```

Comment:
Why is this needed to be explicitly added as dependency? Is this not shipped as a part of bcprov-jdk15on? Wondering how was it working with 1.64

Reply:
bcpkix-jdk15on is a transitive dependency from org.apache.hadoop:hadoop-yarn-server-web-proxy:jar:3.3.1 version, where the bouncy castle version is still 1.60, which has vulnerabilities. hence added it explicitly as dependency with 1.68 version

Reply:
since we are not packaging bcpkix-jdk15on, removed those changes.

@pkumarsinha please help to review the PR